docs/Contributing.asciidoc

```diff
@@ -179,25 +184,90 @@ systemd:: systemd service definitions
 templates:: HTML templates delivered by web UI
 tools:: Development tools
 
+[[dependencies]]
+== Dependencies
+
+Here we differentiate between these types of dependencies:
+
+* Runtime: Run openQA
+* Development:
+** Test: Run openQA's test suite
+** For developers: style checks, code coverage etc.
+
+You can get those dependencies in several ways:
+
+* packaged in openSUSE and other distributions
+* Manually install from source
+* Perl modules via CPAN client
+* Use a prebuilt container
+
+We manage our dependencies via a central `dependencies.yaml` file per
+repository. We generate the `.spec` files for `rpm` as well as a `cpanfile`
+for quicker installation of all perl modules.
+
+[[runtime-dependencies]]
+=== Runtime Dependencies
+
+For openSUSE there is `openQA` and `os-autoinst`, including several subpackages.
+That should be everything necessary to run an openQA instance.
+
+```
+zypper install openQA os-autoinst
+```
+
+If you have to manually install packages, look into the `dependencies.yaml`
+file or the spec file, e.g. `dist/rpm/openQA.spec` (or `dist/rpm/*.spec` in
+general). You can leave out sections starting with `test_requires` for example.
+
+A CPAN client will by default also leave out modules for testing and
+development.
+
+There is a <<Installing.asciidoc#single-instance-container,Single-Instance Container>>
+which can be tried out as a minimal installation.
+
+[[development-dependencies]]
+=== Development Dependencies
+
+For openSUSE:
+```
+zypper install os-autoinst-devel openQA-devel
+```
+
+For distributions without those packages, look into `dependencies.yaml` or
+`dist/rpm/openQA.spec` like above.
+
+The `-devel` packages will install dependencies for tests and development.
+
+If you just want to run the test suite to see if it passes on your platform,
+you would not need the developer's dependencies, like `Perl::Tidy` and
+`Devel::Cover`. They are recommended as soon as you plan a contribution.
+
+For the UI tests you need `perl-Selenium-Remote-Driver` as well as
+`chromedriver` and either `chrome` or `chromium`.
+
+See <<Contributing.asciidoc#run-tests-in-container,Run tests in a container>>
+for a container with all dependencies.
+
 
 [[development-setup]]
 == Development setup
+
+The following explains in detail what is necessary to develop openQA code.
+
+For something to mostly just copy & paste, check out the
```

Comment:
> For something to mostly just copy & paste

I would rather say something like

`The simplest approach. if you are in hurry...`

Reply:
if you need the job done...
Alternative, ...


In general I dont expect _copy paste_ of my words, as they are only for inspiration or demostrate intention. So even without `if you are in hurry` is also ok with me

Reply:
Instead of "For something to mostly just copy & paste" please explain the specific use case this is useful for in comparison to the other existing way. And I assume that the focus of the new section is for system level testing and not for "test driven development", right?

Reply:
I like the current state :+1: